geometry_msgs/Point goal
geometry_msgs/Point orientation
geometry_msgs/Point camera